### PR TITLE
[quant][pt2e] Fix a bug in reference quantized module (decomposed mode)

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #98903

Summary:
Fixed quant_min/quant_max for per channel quantized weight for reference quantized module in decomposed mode,
this bug is triggered while onboard an internal model

Test Plan:
python test/test_quantization.py TestQuantizeFx.test__convert_to_reference_decomposed_fx_per_channel_quant_module

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Review by andrewor14 — APPROVED** [review]

File: test/quantization/fx/test_quantize_fx.py [hunk 1]
```diff
@@ -6158,6 +6159,56 @@ def forward(self, x, weight, bias):
         res = m(*example_inputs)
         self.assertEqual(res, res_ref)
 
+    def test__convert_to_reference_decomposed_fx_per_channel_quant_module(self):
```
Comment (andrewor14) [inline test/quantization/fx/test_quantize_fx.py:6162]: double underscore

Comment (jerryzh168) [reply test/quantization/fx/test_quantize_fx.py:6162]: yeah this is correct since we are testing `_convert_to_reference_decomposed_fx`

**Pull request closed without merging**